Podfile.lock

```diff
@@ -9,6 +9,6 @@ DEPENDENCIES:
 SPEC CHECKSUMS:
   OHHTTPStubs: 3a42f25c00563b71355ac73112ba2324e9e6cef4
 
-PODFILE CHECKSUM: fe73f990cc38a5c9a39ea8dfbe31558c9c18b94a
+PODFILE CHECKSUM: faa0c28f09836bb70ce5e97a28ec7a38a68bb1cb
 
-COCOAPODS: 1.1.0.rc.2
+COCOAPODS: 1.0.1
```

Comment:
Please upgrade to CocoaPods 1.1.0.rc.2. That would obviate the `SWIFT_VERSION` workaround above.